lib/msal-browser/src/request/PopupRequest.ts

```diff
@@ -23,7 +23,8 @@ import { CommonAuthorizationUrlRequest } from "@azure/msal-common";
  * - loginHint                  - Can be used to pre-fill the username/email address field of the sign-in page for the user, if you know the username/email address ahead of time. Often apps use this parameter during re-authentication, having already extracted the username from a previous sign-in using the preferred_username claim.
  * - sid                        - Session ID, unique identifier for the session. Available as an optional claim on ID tokens.
  * - domainHint                 - Provides a hint about the tenant or domain that the user should use to sign in. The value of the domain hint is a registered domain for the tenant.
- * - extraQueryParameters       - String to string map of custom query parameters.
+ * - extraQueryParameters       - String to string map of custom query parameters added to the /authorize call
+ * - tokenQueryParameters       - String to string map of custom query parameters added to the /token call
```

Comment:
Do we have any prior art for the name from other MSALs?

Reply:
I checked .NET, Python, Java, Android and didn't see anything other than `extraQueryParameters`. I think we get to set the precedent :)